modules/core/nuxeo-core-cache/src/main/java/org/nuxeo/ecm/core/cache/CacheServiceImpl.java

```diff
@@ -189,7 +183,7 @@ public int getApplicationStartedOrder() {
     public void start(ComponentContext context) {
         super.start(context);
         ClusterService clusterService = Framework.getService(ClusterService.class);
-        if (clusterService.isEnabled()) {
+        if (clusterService != null && clusterService.isEnabled()) {
```

Comment:
If `clusterService` is `null` then we have a problem because the point of these caches is to have cluster-wide invalidations... Except maybe in unit tests but I'd rather have the normal code changed just for unit tests that don't initialize all they should

Reply:
It was indeed in unit tests -- can't remember which right now but will lookup the stack trace in next validation tests output.

Reply:
Actually TestCacheDescriptor is the one running into this issue: will adapt the test features.